### PR TITLE
Minor bug fix for ownership tab change stale banner wording

### PR DESCRIPTION
**Pull request by cptran777**
Fix issue where typeahead in ownership tab losing focus would incorrectly display placeholder without a clear. Also fix padding issues on no owners suggested state for suggested owners box. Make change to wording banner for stale search.

`ember test` results:
```
1..364
# tests 364
# pass  358
# skip  6
# fail  0

# ok
```

### CODE REVIEW

File: wherehows-web/app/templates/components/notifications/partials/_stale-search-alert.hbs [hunk 1]
```diff
@@ -1,2 +1,9 @@
-The WhereHows search index isn't up to date. Until that gets fixed, there are two workarounds: <br>
-Search for your dataset on CMON (go/cmon) and click on the browse in WhereHows link OR use the browse view in Wherehows to navigate the dataset tree to your dataset.
+The WhereHows search index is currently out of date.<br>
+Please see
+<a
+ href="https://iwww.corp.linkedin.com/wiki/cf/display/ENGS/Metadata+Annotation+FAQs#MetadataAnnotationFAQs-stale-index"
```
Comment (theseyi) [inline wherehows-web/app/templates/components/notifications/partials/_stale-search-alert.hbs]: Can we create go links instead? This is opensource. Ideally, this should all come from an api, I'll create a task for myself for the next sprint. In the interim, go links are preferred

File: wherehows-web/app/components/pwr-user-lookup.ts [hunk 2]
```diff
@@ -64,7 +64,7 @@ export default class PowerUserLookup extends UserLookup {
   focusOut() {
     setProperties(this, {
       suggestedText: '',
-      showPlaceholder: true
+      showPlaceholder: get(this, 'selectedEntity') ? false : true
```
Comment (theseyi) [inline wherehows-web/app/components/pwr-user-lookup.ts]: `!get(this, 'selectedEntity')`

File: wherehows-web/app/templates/components/notifications/partials/_stale-search-alert.hbs (hunk 1 above)
Comment (mars-lan) [reply wherehows-web/app/templates/components/notifications/partials/_stale-search-alert.hbs]: go/metadata/faq#MetadataAnnotationFAQs-stale-index does the trick

**Comment by cptran777** [comment]
Requested changes made, going ahead with merge.

**Pull request merged**